src/main/scala/com/raphtory/algorithms/MotifAlpha.scala

```diff
@@ -2,12 +2,18 @@ package com.raphtory.algorithms
 
 import com.raphtory.core.model.algorithm.{GraphAlgorithm, GraphPerspective, Row}
 
-/*
-
-This is an implementation of the alpha_1 motif from the paper
-Detecting Mixing Services via Mining Bitcoin Transaction Network With Hybrid Motifs
- */
-
+/**
+Description
+  The algorithms identifies 2-edge-1-node temporal motifs; It detects one type of motifs:
+  For each vertex, for each incoming edge it checks whether there are any outgoing edges that occur after
```

Comment:
For each incoming edge a vertex has, the algorithm checks whether there are any outgoing edges which occur after it and returns a count of these.